Dockerfile

```diff
@@ -204,23 +204,27 @@ ENV \
 
 WORKDIR /root/
 
-# Precaching atlases + AtlasPack
+# Precaching templates
+COPY scripts/fetch_templates.py fetch_templates.py
+RUN ${CONDA_PYTHON} -m pip install --no-cache-dir --upgrade templateflow && \
```

Comment:
I think templateflow is already installed in the conda python [here](https://github.com/PennLINC/qsiprep_build/blob/main/qsiprep_env.yml#L38). Was it not showing up as available in the build?

Reply:
Sorry, I just copied over this section from the fMRIPrep Dockerfile. I'll drop the install.

Reply:
I'm downloading a copy of qsirecon_build just to make sure templateflow is there

Reply:
it's in there

Reply:
Sweet! I'll resolve this then. Any other concerns about the PR?